pkg/apis/pipeline/v1alpha1/pull_request_resource.go

```diff
@@ -55,16 +57,23 @@ func NewPullRequestResource(prImage string, r *PipelineResource) (*PullRequestRe
 		return nil, fmt.Errorf("PipelineResource: Cannot create a PR resource from a %s Pipeline Resource", r.Spec.Type)
 	}
 	prResource := PullRequestResource{
-		Name:    r.Name,
-		Type:    r.Spec.Type,
-		Secrets: r.Spec.SecretParams,
-		PRImage: prImage,
+		Name:      r.Name,
+		Type:      r.Spec.Type,
+		Secrets:   r.Spec.SecretParams,
+		PRImage:   prImage,
+		SSLVerify: true,
 	}
 	for _, param := range r.Spec.Params {
 		if strings.EqualFold(param.Name, "URL") {
 			prResource.URL = param.Value
 		} else if strings.EqualFold(param.Name, "Provider") {
 			prResource.Provider = param.Value
+		} else if strings.EqualFold(param.Name, "SSLVerify") {
+			verify, err := strconv.ParseBool(param.Value)
+			if err != nil {
+				return nil, fmt.Errorf("PipelineResource: Error occurred converting %s to boolean in Pipeline Resource %s", param.Value, r.Name)
```

Comment:
Suggest removing the "PipelineResource: " prefix in the message here.  "Pipeline Resource" is mentioned again later in the same string.

Suggest changing `%s` to `%q` for the param.Value just in case it's an empty string - it'll render as `""` instead of a missing character.

Also, please lowercase the entire error message.

Reply:
@sbwsg Would you like the error on line 57 changed as well?  I copied the format from there.

Reply:
:+1: thanks @dibbles !